cg/apps/lims/api.py

```diff
@@ -1,7 +1,7 @@
 """Contains API to communicate with LIMS"""
 import datetime as dt
```

Comment:
This file has only black changes. Or am i missing something?

Reply:
Yes I think Maria is applying the suggestion from sorcery to all files that she modifies

Reply:
YUP and it looks so much better